src/System.Private.CoreLib/shared/System/Type.cs

```diff
@@ -165,9 +166,9 @@ public MethodInfo GetMethod(string name, BindingFlags bindingAttr)
         }
 
         public MethodInfo GetMethod(string name, Type[] types) => GetMethod(name, types, null);
-        public MethodInfo GetMethod(string name, Type[] types, ParameterModifier[] modifiers) => GetMethod(name, Type.DefaultLookup, null, types, modifiers);
-        public MethodInfo GetMethod(string name, BindingFlags bindingAttr, Binder binder, Type[] types, ParameterModifier[] modifiers) => GetMethod(name, bindingAttr, binder, CallingConventions.Any, types, modifiers);
-        public MethodInfo GetMethod(string name, BindingFlags bindingAttr, Binder binder, CallingConventions callConvention, Type[] types, ParameterModifier[] modifiers)
+        public MethodInfo GetMethod(string name, Type[] types, ParameterModifier[]? modifiers) => GetMethod(name, Type.DefaultLookup, null, types, modifiers);
```

Comment:
All overrides of `GetMethod` can return a `null` MethodInfo. https://source.dot.net/#System.Private.CoreLib/src/System/RtType.cs,2738

Reply:
yeah, its coming with MethodInfo nullable, for current context it is not non null